src/Facebook/GraphNodes/GraphGroup.php

```diff
@@ -0,0 +1,171 @@
+        'venue' => '\Facebook\GraphNodes\GraphLocation',
+    ];
+
+    /**
+     * Returns the `id` (The Group ID) as string if present.
+     *
+     * @return string|null
+     */
+    public function getId()
+    {
+        return $this->getField('id');
+    }
+
+    /**
+     * Returns the `cover` (The cover photo of the Group) as GraphCoverPhoto if present.
+     *
+     * @return GraphCoverPhoto|null
+     */
+    public function getCover()
+    {
+        return $this->getField('cover');
+    }
+
+    /**
+     * Returns the `description` (A brief description of the Group) as string if present.
+     *
+     * @return string|null
+     */
+    public function getDescription()
+    {
+        return $this->getField('description');
+    }
+
+    /**
+     * Returns the `email` (The email address to upload content to the Group. Only current members of the Group can use this) as string if present.
+     *
+     * @return string|null
+     */
+    public function getEmail()
+    {
+        return $this->getField('email');
+    }
+
+    /**
+     * Returns the `icon` (The URL for the Group's icon) as string if present.
+     *
+     * @return string|null
+     */
+    public function getIcon()
+    {
+        return $this->getField('icon');
+    }
+
+    /**
+     * Returns the `link` (The Group's website) as string if present.
+     *
+     * @return string|null
+     */
+    public function getLink()
+    {
+        return $this->getField('link');
+    }
+
+    /**
+     * Returns the `name` (The name of the Group) as string if present.
+     *
+     * @return string|null
+     */
+    public function getName()
+    {
+        return $this->getField('name');
+    }
+
+    /**
+     * Returns the `member_request_count` (Number of people asking to join the group.) as int if present.
+     *
+     * @return int|null
+     */
+    public function getMemberRequestCount()
+    {
+        return $this->getField('member_request_count');
+    }
+
+    /**
+     * Returns the `owner` (The profile that created this Group) as graphNode if present.
+     *
+     * @return graphNode|null
+     */
+    public function getOwner()
+    {
+        return $this->getField('owner');
+    }
+
+    /**
+     * Returns the `parent` (The parent Group of this Group, if it exists) as graphNode if present.
+     *
+     * @return graphNode|null
+     */
+    public function getParent()
+    {
+        return $this->getField('parent');
+    }
+
+    /**
+     * Returns the `privacy` (The privacy setting of the Group) as string if present.
+     *
+     * @return string|null
+     */
+    public function getPrivacy()
+    {
+        return $this->getField('privacy');
+    }
+
+    /**
+     * Returns the `updated_time` (The last time the Group was updated (this includes changes in the Group's properties and changes in posts and comments if user can see them)) as DateTime if present.
+     *
+     * @return DateTime|null
```

Comment:
My IDE (phpStorm) can't figure out the reference since the file is namespaced. So it tries to find `Facebook\GraphNodes\DateTime`